core-bundle/src/Event/ContaoCoreEvents.php

```diff
@@ -68,6 +68,15 @@ final class ContaoCoreEvents
      */
     public const PREVIEW_URL_CONVERT = 'contao.preview_url_convert';
 
+    /**
+     * The contao.robots_txt event is triggered when the /robots.txt route is called.
+     *
+     * @var string
+     *
+     * @see RobotsTxtEvent
+     */
+    public const ROBOTS_TXT = 'contao.robots_txt';
```

Comment:
It is deprecated in Symfony to use event names. The event class should be the name…

Reply:
Yes but I cannot just change that atm as we do not require 4.4 yet.

Reply:
why not? Just use `event::class` as the event name?